yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@babel/cli@^7.2.3":
   version "7.13.14"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.13.14.tgz#c395bc89ec4760c91f2027fa8b26f8b2bf42238f"
+  resolved "https://registry.npmjs.org/@babel/cli/-/cli-7.13.14.tgz"
```

Comment:
revert the changes in `yarn.lock`, it for some reason change registry to npmjs in your PR

Reply:
Thank you for leaving your reviews!! @jeetiss 
We'll send you the PR again after making the revisions to this as soon as possible.

And can we get another good first issue? We want to contribute more to this repository!